lib/FrontendTool/FrontendTool.cpp

```diff
@@ -262,6 +264,318 @@ template <> struct ObjectTraits<LoadedModuleTraceFormat> {
 }
 }
 
+static bool isClangOverlayOf(ModuleDecl *potentialOverlay,
+                             ModuleDecl *potentialUnderlying) {
+  return !potentialOverlay->isNonSwiftModule()
+      && potentialUnderlying->isNonSwiftModule()
+      && potentialOverlay->getName() == potentialUnderlying->getName();
+}
+
+// TODO: Delete this once changes from https://reviews.llvm.org/D83449 land on
+// apple/llvm-project's swift/master branch.
```

Comment:
Has this landed yet?

Reply:
No it hasn't, I will cherry-pick those changes after the rebranch (the rebranch happened before the changes landed).